### PR TITLE
Updated link of Our Services in footer

### PR DESCRIPTION
**Pull request by ShubhaSwarup**
## Description 📝
Updated link of Our Services in footer 

Fixes: <link>#253
[Updated link of Our Services in footer]
## Changes 💬

- High-level overview of the changes introduced.
- Step-by-step breakdown of the implementation, if applicable.

## Checklist ✅

- [✅] Reviewed the code changes thoroughly.
- [ ] Tested the changes locally and ensured they work as expected.
- [✅ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [ ✅] Code adheres to the project's style guidelines.
- [✅ ] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)

Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

### CODE REVIEW

**Comment by PiyushKalyanpy** [comment]
Please update the issue link in the description

File: Components/LandingFooter.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const LandingFooter = () => {
                 <Link href="/courses" className={hoverEffect}>Courses</Link>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                <Link href="/OurServices" className={hoverEffect}>Our Services</Link>
+                <Link href="/OurServices.txs" className={hoverEffect}>Our Services</Link>
```
Comment (lalitmee) [inline Components/LandingFooter.tsx:28]: Is this a valid URL?

**Review by lalitmee — CHANGES_REQUESTED** [review]
The PR description formatting is not correct. Please fix that too.

**Comment by PiyushKalyanpy** [comment]
Hey there, 
We are closing this pull request as the changes we requested were not implemented. Please create a new PR once you implement the changes.

**Pull request closed without merging**